dist/stylesheets/css/mindsmash-ui.css

```diff
@@ -3447,3 +3457,35 @@ dl dd { margin-bottom: 12px; }
 .dl-horizontal dt { clear: left; float: left; margin-right: 24px; text-align: right; width: calc(35% - 12px); }
 
 .dl-horizontal dd { margin-left: 0; width: calc(65% - 12px); }
+
+.navbar.navbar-msm { background-color: #317DC1; color: #FFF; height: 60px; border: 0; }
+
+.navbar.navbar-msm a { color: #FDFDFD; }
+
+.navbar.navbar-msm .navbar-brand { height: 60px; font-size: 24px; padding: 18px 20px; }
+
+.navbar.navbar-msm .navbar-toggle { border: 1px solid rgba(255, 255, 255, 0.5); margin: 13px 20px; }
+
+.navbar.navbar-msm .navbar-toggle .icon-bar { background-color: rgba(255, 255, 255, 0.5); }
+
+.navbar.navbar-msm .navbar-toggle:hover, .navbar.navbar-msm .navbar-toggle:focus { border-color: #FFF; }
+
+.navbar.navbar-msm .navbar-toggle:hover .icon-bar, .navbar.navbar-msm .navbar-toggle:focus .icon-bar { background-color: white; }
+
+.navbar.navbar-msm ul.nav li > a { padding: 18px 20px; color: rgba(255, 255, 255, 0.75); }
+
+.navbar.navbar-msm ul.nav li > a:hover, .navbar.navbar-msm ul.nav li > a:focus { background-color: rgba(0, 0, 0, 0.1); color: #FFF; }
```

Comment:
Multiple classes and nesting in css preprocessors lead to specificity problems. In case you want to override the padding here, you now need three classes or an `!important` statement. Let's try to avoid this as mindsmash-ui should be easy extendable even though it is somewhat closed for changes.